device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/TransportException.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.device.transport;
+
+import com.microsoft.azure.sdk.iot.device.IotHubStatusCode;
+import com.microsoft.azure.sdk.iot.device.exceptions.IotHubClientException;
+import com.microsoft.azure.sdk.iot.device.exceptions.MultiplexingClientRegistrationException;
+import com.microsoft.azure.sdk.iot.device.transport.amqps.exceptions.*;
+import com.microsoft.azure.sdk.iot.device.transport.https.exceptions.InternalServerErrorException;
+import com.microsoft.azure.sdk.iot.device.transport.https.exceptions.ServerBusyException;
+import com.microsoft.azure.sdk.iot.device.transport.https.exceptions.ThrottledException;
+import com.microsoft.azure.sdk.iot.device.transport.https.exceptions.UnauthorizedException;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt.exceptions.MqttBadUsernameOrPasswordException;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt.exceptions.MqttIdentifierRejectedException;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt.exceptions.MqttServerUnavailableException;
+import com.microsoft.azure.sdk.iot.device.transport.mqtt.exceptions.MqttUnauthorizedException;
+
+import static com.microsoft.azure.sdk.iot.device.transport.TransportException.IotHubService.NOT_APPLICABLE;
+
+public class TransportException extends Exception
+{
+    protected boolean isRetryable = false;
+
+    public enum IotHubService
```

Comment:
The name seems too generic to me. What about IotHubOperation?

Reply:
I'll delete this enum as per the above